lib/web_ui/README.md

```diff
@@ -27,3 +177,58 @@ directly), follow these steps to roll to the new version:
 
 If you have questions, contact the Flutter Web team on Flutter Discord on the
 #hackers-web-🌍 channel.
+
+### Configuration files
+
+`browser_lock.yaml` contains the version of browsers we use to test Flutter for
+web. Versions are not automatically updated whenever a new release is available.
+Instead, we update this file manually once in a while.
+
+`goldens_lock.yaml` refers to a revision in the https://github.com/flutter/goldens
+repo. Screenshot tests are compared with the golden files at that revision.
+When making engine changes that affect screenshots, first submit a PR to
+flutter/goldens updating the screenshots. Then update this file pointing to
+the new revision.
+
+`canvaskit_lock.yaml` locks the version of CanvasKit for tests and production
+use.
+
+## Troubleshooting
+
+### Can't load Kernel binary: Invalid kernel binary format version.
+
+Sometimes `.dart_tool` cache invalidation fails, and you'll end up with a
+cached version of `felt` that is not compatible with the Dart SDK that you're
+using.
+
+In that case, any invocation to `felt` will fail with:
+
+```
+Can't load Kernel binary: Invalid kernel binary format version.
+```
+
+The solution is to delete the cached `felt.snapshot` files under `lib/web_ui`:
+
+```
+rm .dart_tool/felt.snapshot*
```

Comment:
I now use: 

```
FELT_USE_SNAPSHOT=false felt ...
```

That usually fixes the tool from that moment on.

Reply:
That fixes the kernel mismatch issue but it disables the snapshot, which slows down felt start-up time:

```
$ time felt test test/alarm_clock_test.dart 
real	0m10.601s
user	0m12.223s
sys	0m2.359s

$ FELT_USE_SNAPSHOT=false time felt test test/alarm_clock_test.dart
real	0m21.578s
user	0m27.702s
sys	0m4.177s
```

So I wouldn't recommend it.